lms/static/coffee/src/navigation.js

```diff
@@ -0,0 +1,70 @@
+var edx = edx || {},
```

Comment:
Ah, that's why I'm not seeing any issues on your branch! I'll remove the file and try again...